apps/fortuna/src/keeper.rs

```diff
@@ -837,3 +846,68 @@ pub async fn track_provider(
         })
         .set(end_sequence_number as i64);
 }
+
+// TODO: fields
```

Comment:
no fields necessary here. We only want the `chain_id` and if you add `in_current_span` when you are spawning this, it should be enough.